vagrant/ansible/roles/report.versions/tasks/main.yml

```diff
@@ -10,7 +10,16 @@
   when: ansible_facts.packages[item] is defined
   loop:
     - kernel
-    - samba-client
     - samba
-    - ctdb
     - glusterfs
+
+- name: Extract Ceph version
+  block:
+    - name: Run 'cephadm version'
+      shell: /root/cephadm version | awk -F"[()]" '{print $2}'
```

Comment:
I've checked in my test machine and cephadm and ceph are the same version, but given how we install it, I'm not sure if this is guaranteed.

Reply:
I assume they are directly related i.e, the `cephadm` version used to setup a cluster will always match the version of Ceph that gets installed on the nodes. In case of any mismatch I expect a failure in cluster setup.

@phlogistonjohn can you please confirm?

Reply:
That's true unless you manually specify the image (`--image` option) to use during bootstrap. Otherwise whatever is "hard coded" in the cephadm sources will be used and that's generally going to match the build (main=>main, reef=>reef, etc).

Reply:
My concern is about the exact hash (commit) we are testing. Cephadm is installed from a repo, while the containers are installed from an image in quay. I'm not sure if it's guaranteed that the repo and the quay image are always built from the same exact commit.

In case of a problematic patch, having it or not may be significant, and we should know it for sure.

Reply:
How about running `cephadm shell -- ceph version` instead of `cephadm version`? This will be more helpful I guess.

Reply:
I think that the best solution could be to show the version of cephadm **and** ceph.

Reply:
OK. Updated.